### PR TITLE
eduki.com

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/122020

<details><summary>Screenshot:</summary>

Снимок экрана 2022-06-20 в 13 30 37

</details><br/>

### CODE REVIEW

**Comment by Alex-302** [comment]
Missed analytics and tracking cookie
<details><summary>Screenshot:</summary>

image

image

</details>

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -52,6 +52,7 @@
 !
 ||cc.loginfra.com/cc?a=$image
 ||cc.naver.com/cc$image,other,ping
+||api.eduki.com/api/v2/events/track
```
Comment (zloyden) [inline SpywareFilter/sections/specific.txt:55]: `v2` is not necessary.
`||api.eduki.com/api/*/events/track` or `||api.eduki.com/*/track`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply SpywareFilter/sections/specific.txt:55]: And cookie is still not blocked

Comment (zloyden) [reply SpywareFilter/sections/specific.txt:55]: Yep, what about this rule?
```
eduki.com##.wrap > div.modal-bg
```

**Review by zloyden — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**